src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java

```diff
@@ -54,6 +59,7 @@ public enum ThePrayer implements HasLabel {
     private static final Map<String, TriConsumer<Map<String, Object>, String, Object>> migrations = Map.of(
         "exampleRetiredField", moveTo("applicant1FirstName"),
         "solServiceMethod", moveTo("serviceMethod"),
+        "applicant1ScreenHasMarriageBroken", migrateMarriageBroken(),
```

Comment:
don't we also need applicant2 screen broken field for joint applications ?

Reply:
Spoke to Lalita about this and the outcome was another ticket eventually to do this functionality.